app/[lang]/page.tsx

```diff
@@ -3,12 +3,14 @@ import React from 'react';
 import LiatoshynskyOffice from '~/components/Liatoshynsky-office/LiatoshynskyOffice';
 
 import FoundationFounders from '~/shared/components/FoundationFounders/FoundationFounders';
+import AboutFoundation from '~/shared/components/main-page-sections/about-foundation/AboutFoundation';
 
 export default async function Home() {
   return (
     <>
-      <LiatoshynskyOffice />
+      <AboutFoundation />
       <FoundationFounders />
+      <LiatoshynskyOffice />
```

Comment:
```suggestion
      <AboutFoundation />
       <LiatoshynskyOffice />
      <FoundationFounders />
```
If I remember correctly, the order in figma is like this

Reply:
updated with new added section as well